scannerazure.properties

```diff
@@ -6,8 +6,13 @@ organizationUrl=https://www.sonarsource.com
 homepageUrl=https://docs.sonarqube.org/latest/analysis/scan/sonarscanner-for-azure-devops/
 issueTrackerUrl=https://sonarsource.atlassian.net/jira/software/c/projects/SONARAZDO/issues
 sourcesUrl=https://github.com/SonarSource/sonar-scanner-azdo
-archivedVersions=4.7,4.7.1,4.7.2,4.8,4.8.1,4.9,4.10,4.11,4.12,4.16,4.17,4.18,4.19,4.20,4.21,4.22,4.23,4.23.1,5.0.0,5.1.0,5.1.1,5.2.0,5.3.0,5.4.0,5.5.0,5.6.0,5.6.1,5.7.0,5.8.0,5.8.1,5.9.0,5.10.0,5.11.0,5.11.1,5.12.0,5.13.0,5.14.0,5.15.0,5.16.0,5.17.2,5.18.2,5.18.3,5.18.4,5.19.0,5.19.1,5.19.2,5.20.0,6.0.0,6.0.1,6.1.0
-publicVersions=6.2.0
+archivedVersions=4.7,4.7.1,4.7.2,4.8,4.8.1,4.9,4.10,4.11,4.12,4.16,4.17,4.18,4.19,4.20,4.21,4.22,4.23,4.23.1,5.0.0,5.1.0,5.1.1,5.2.0,5.3.0,5.4.0,5.5.0,5.6.0,5.6.1,5.7.0,5.8.0,5.8.1,5.9.0,5.10.0,5.11.0,5.11.1,5.12.0,5.13.0,5.14.0,5.15.0,5.16.0,5.17.2,5.18.2,5.18.3,5.18.4,5.19.0,5.19.1,5.19.2,5.20.0,6.0.0,6.0.1,6.1.0,6.2.0
+publicVersions=6.3.1
+
+6.3.1.description=Embed Scanner for .NET 6.2.0.85879
```

Comment:
Is there anything we can say about what it brings? Mostly bug fixes from what I see.

Reply:
6.3.0 was supposed to bring "JVM no longer required".
6.3.1 reverted this change because of a bug in the Scanner .NET 7.0